dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequest.java

```diff
@@ -4308,6 +4274,18 @@ protected String getSrtHelperCharEncoding() {
         return _srtRequestHelper._characterEncoding;
     }
     
+    protected void srtRequestDataInitNull() {
+        this._requestURI = null;
+        this._pathInfo = null;
+        this._queryString=null;
+        this._parameters=null;
+        this._queryStringList = null;
+        this._qsSetExplicit = false;
+        if (!_paramStack.isEmpty()) {
+            _paramStack.clear();
+        }
```

Comment:
paramStack is not a new field, so not sure if you should be messing with it.

Reply:
That was copied over from the init method in SRTServletRequestThreadData ... this is one of two fields that existed in both SRTServletRequest and the threadlocal data.

You are right that _paramStack is not reset/cleared in SRTServletRequest.initForNextRequest() ... that seems wrong, should anything left on the paramStack really be carried over from request to request?

I am generally confused about why the original implementor would choose to maintain two copies of the same field, in somewhat different ways, when the only users of the fields are in SRTServletRequest.

Reply:
agree, have removed the paramStack parts of srtRequestDataInitNull